src/Interpreters/executeDDLQueryOnCluster.h

```diff
@@ -43,7 +44,7 @@ struct DDLQueryOnClusterParams
 BlockIO executeDDLQueryOnCluster(const ASTPtr & query_ptr, ContextPtr context, const DDLQueryOnClusterParams & params = {});
 
 BlockIO getDistributedDDLStatus(
-    const String & node_path, const DDLLogEntry & entry, ContextPtr context, const std::optional<Strings> & hosts_to_wait = {});
+    const String & node_path, const DDLLogEntry & entry, ContextPtr context, const std::optional<Strings> & hosts_to_wait);
```

Comment:
Not related that much to PR but `const std::optional<Strings> & ` is bad, better do `const Strings * hosts_to_wait` so you don't have to copy to `std::optional` if you have list of hosts.